changelog/bugfixes/2024-05-31-disable-dhcp-rapid-commit-for-cloudsigma.md

```diff
@@ -0,0 +1 @@
+- Disabled the new DHCP RapidCommit feature which is enabled by default since systemd 255. CloudSigma provides an incompatible implementation which results in cloud-init not being applied as no IP is issued. ([Flatcar#1438](https://github.com/flatcar/Flatcar/issues/1438))
```

Comment:
Hello, the link in the changelog should be to this PR and the commit message can contain something like:

Fixes: <issue link>
or
See: <issue link>

Reply:
Thanks, i adapted it. Hopefully the way you meant :).